src/apps/common/submodules/navigationWizard/navigationWizard.js

```diff
@@ -207,19 +207,23 @@ define(function(require) {
 					});
 
 			$template
-				.on('click', '#done:not(.disabled)', function(event) {
-					event.preventDefault();
+				.find('#done')
```

Comment:
Checks against disabled pseudo-classes can be achieve directly at the selector level:
```suggestion
				.find('#done:enabled')
```

Reply:
Yes, but the element is looked for once before it is disabled. However, the click event is not triggered if the button is disabled, so I removed the extra check in the handler function.